src/mlpack/methods/lmnn/lmnn.hpp

```diff
@@ -77,9 +77,12 @@ class LMNN
    * LearnDistance() is called with an outputMatrix with correct dimensions,
    * then that matrix will be used as the starting point for optimization.
    *
+   * @tparam CallbackTypes Types of Callback functions.
    * @param outputMatrix Covariance matrix of Mahalanobis distance.
+   * @param callbacks Callback Functions.
```

Comment:
It might be useful to add a comment clarifying what these callbacks are for, e.g.,

```
Callback functions for ensmallen optimizer `OptimizerType`.  See <link to ensmallen callback documentation>.
```

Reply:
Done :)